City-of-Helsinki/helsinki-paatokset
# PP-243: fixed navigation issues, updated hdbt to newer version

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-243

Updated hdbt to newer version, this lead to icon name change so I had to change all icons from hds-icon to hel-icon, added side navigation styles also to normal pages and added navigation js in preprocessing.

**How to test**

Checkout branch and run `make composer-install` and `make drush-cim`.

Check that the icons still work around the site. i.ex. on https://helsinki-paatokset.docker.so/fi/paattajat page in the listing.

Test the main navigation functionality by creating a page that has other (normal)pages as children. Add the parent page to main navigation and check that the hover effect works on it (it should show the children in the hover menu). 

Check that if a normal page has a parent page and another page as a "sibling". The styles in the side navigation should be correct.
